test/testsuites/opsrcpropagationtests.go

```diff
@@ -0,0 +1,73 @@
+package testsuites
+
+import (
+	"testing"
+
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	helpers "github.com/operator-framework/operator-marketplace/test/helpers"
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	apps "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// OPSRCPropagation is a test suit that confirms that the expected kubernetets resources are
+// created by marketplace after the creation of an operatorsource.
+func OPSRCPropagation(t *testing.T) {
```

Comment:
Maybe this should be capitalized like `OpSrcPropagation`? We have that in other places in the source.

Reply:
I'm happy to make this change.